### PR TITLE
add jdot project

### PR DESCRIPTION
**Pull request by gforsyth**
Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml".
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/5eddbd7fc9d1502169089da06c3688d9759be978/recipes/example/meta.yaml#L64-L73) for an example).
- [x] Source is from official source.
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged).
- [x] If static libraries are linked in, the license of the static library is packaged.
- [x] Build number is 0.
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details).
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there.
- [x] When in trouble, please check our [knowledge base documentation](https://conda-forge.org/docs/maintainer/knowledge_base.html) before pinging a team.

### CODE REVIEW

**Comment by gforsyth** [comment]
ping @anjakefala and @saulpw -- I've added you both as package maintainers so that y'all can approve new releases, but if you don't need/want the extra noise that's no problem at all, just want to make sure you have access if you want it.  (Please do respond when time allows in affirmative or negative so we can get this pushed up.)  :pray:

**Comment by anjakefala** [comment]
@gforsyth I am in the "joyfully affirmative" camp!

File: recipes/jdot/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+    - python >=3.7
+  run:
+    - python >=3.7
+
+test:
+  imports:
+    - jdot
+  commands:
+    - pip check
+    - jdot --help
```
Comment (anjakefala) [inline recipes/jdot/meta.yaml:32]: This does not exist yet. :disappointed: https://github.com/saulpw/jdot/issues/10

Maybe this is the pressure cooker to implement it..

**Comment by saulpw** [comment]
Less noise for me, thank you for asking.

File: recipes/jdot/meta.yaml (hunk 1 above)
Comment (anjakefala) [reply recipes/jdot/meta.yaml:32]: It does [now](https://github.com/saulpw/jdot/pull/13)!

**Pull request merged**